src/components/regions/Regions.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react';
+import { shallowEqual, useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import Pagination from 'react-bootstrap/Pagination';
+import Region from './Region';
+import Header from '../Header';
+import Loading from '../Loading';
+import { fetchCountriesAction } from '../../redux/countries/countries';
+
+const Regiones = () => {
+  const { id } = useParams();
+  const dispatch = useDispatch();
+  const { countries, loaded } = useSelector((state) => state.countries, shallowEqual);
+  const [regions, setRegions] = useState(null);
+  const [activePage, setActivePage] = useState(1);
+
+  useEffect(() => {
+    dispatch(fetchCountriesAction());
+    if (loaded) {
+      const countryObj = countries.filter((obj) => obj.id === id);
+      // Non camelCase are needed here since they're used by the Narrativa API.
+      /* eslint-disable camelcase */
```

Comment:
- [ ] please kindly avoid using several eslint-disable comments to get around ESlint errors. these ESlint errors are at best showing that you are not adhering to some coding standards or at worst highlighting a possible underlying issue with your code. please kindly address these issues.
- [ ] in these case you rename the variables that you get from the Narrativa API to have camelCase